app/javascript/components/requests/axios_requests.js

```diff
@@ -40,4 +40,9 @@ export const apiRequest = async ( method, data, setData, redirect = ()=>{}, url
       break
     default:
   }
+}
+
+export const signOutUser = async () =>{
+  createCsrfToken()
+  await axios.get('/api/v1/sign_out_user').then(()=>console.log("success sign out"))
```

Comment:
Is this console log something we want in the final product? If so, please explain why.